package.json

```diff
@@ -25,7 +29,9 @@
     }
   },
   "jest": {
-    "preset": "@theholocron/jest-preset"
+    "preset": "@theholocron/jest-preset",
+    "testEnvironment": "jest-environment-node",
+    "transform": {}
```

Comment:
> Ensure you either disable code transforms by passing transform: {} or otherwise configure your transformer to emit ESM rather than the default CommonJS (CJS).

- https://jestjs.io/docs/ecmascript-modules